README.md

```diff
@@ -101,21 +101,32 @@ shape: (5, 8)
 
 ## Performance 🚀🚀
 
+### Blazingly fast
+
 Polars is very fast. In fact, it is one of the best performing solutions available.
 See the results in [h2oai's db-benchmark](https://h2oai.github.io/db-benchmark/).
 
 In the [TPCH benchmarks](https://www.pola.rs/benchmarks.html) polars is orders of magnitudes faster than pandas, dask, modin and vaex
 on full queries (including IO).
 
-Besides fast, polars is also very lightweight. It comes with zero required dependencies, and this shows in the import times:
+### Lightweight
 
-### import time measurements:
+Polars is also very lightweight. It comes with zero required dependencies, and this shows in the import times:
 
 - polars: 70ms
 - numpy: 104ms
 - pandas: 520ms
 
-## Python setup
+### Handles larger than RAM data
+
+If you have data that does not fit into memory, polars lazy is able to process your query (or parts of your query) in a
+streaming fashion, this drastically reduces memory requirements you might be able to process your 250GB dataset on your
+laptop. Collect with `collect(allow_streaming=True)` to run the query streaming. (This might be a little slower, but
```

Comment:
Isn't it `.collect(streaming=True)` now and not `collect(allow_streaming=True)`?